SemanticCompoundQueries.php

```diff
@@ -17,7 +17,7 @@
 
 if ( !defined( 'MEDIAWIKI' ) ) die();
 
-define( 'SCQ_VERSION', '0.4.1' );
+define( 'SCQ_VERSION', '1.0' );
```

Comment:
Well, I'm aware of the Semver reading and as I pointed out do as you see fit. I don't have a big investment here but of course I'd like to see quality software being released and 1.0 would signal this and in my opinion this repo is not of the quality I'd like to promote.